### DISCUSSION TITLE
NAR M1 Membership API

### DISCUSSION THREAD

**Discussion by gr33neggs** [RESOStandards/uli-service#19] [General]
I posted this on the R and D confluence, but it probably belongs here.

Has NAR’s M1 API been looked into to see if it's relevant to ULI?

[Welcome to M1 Tech Center](https://nationalasnofrealtors.mcoutput.com/m1techcenter/Content/Home.htm) 

NAR describes M1 as “a powerful member information engagement platform, intended to be the single source of truth for clean, accurate, and actionable member data.“

[REALTORS® M1: Member Engagement System](https://www.nar.realtor/ae/manage-your-association/realtors-m1-members-first-engagement-system) 

The above description is why I bring this up. If NAR intends it to be the source for member data, I think what ever ULI implementation is decided on (I am guessing Decentralized) may need to include it as a source. The only issue is that the M1 API isn’t OData based, though it is a JSON payload at least. 

The M1 API is a big topic for the AMS side, as NAR is really pushing for all associations to have it implemented by summer 2024 and support for the old CDB process goes away August 2023.

**Comment by darnjo** [comment]
Thanks, Ryan. 

M1 covers a lot of the members in the U.S. but not everyone. One of the project requirements is that it could support licensees outside of the NAR. 

That said, the [MemberNationalAssociationId](https://ddwiki.reso.org/display/DDW17/MemberNationalAssociationId+Field) plays a significant role in the ULI matching formula, when present. From the DD Wiki Usage Stats, it looks like about 70% of (mostly MLS) orgs have it, and 75% of systems. This doesn't include AMSs, as we haven't certified any yet.
